tests/common/test_data_writers/test_parquet_writer.py

```diff
@@ -0,0 +1,164 @@
+        writer.write_data_item([{"col1": 1, "col2": 2, "col3": {"hello":"marcin"}}], {"col1": c1, "col2": c2, "col3": c3})
+
+    with open(writer.closed_files[0], "rb") as f:
+        table = pq.read_table(f)
+        assert table.column("col1").to_pylist() == [1, 1]
+        assert table.column("col2").to_pylist() == [2, 2]
+        assert table.column("col3").to_pylist() == ["""{"hello":"dave"}""","""{"hello":"marcin"}"""]
+
+
+def test_parquet_writer_all_data_fields() -> None:
+
+    # can be reused for other writers..
+    columns = {
+        "col1": new_column("col1", "text"),
+        "col2": new_column("col2", "double"),
+        "col3": new_column("col3", "bool"),
+        "col4": new_column("col4", "timestamp"),
+        "col5": new_column("col5", "bigint"),
+        "col6": new_column("col6", "binary"),
+        "col7": new_column("col7", "complex"),
+        "col8": new_column("col8", "decimal"),
+        "col9": new_column("col9", "wei"),
+        "col10": new_column("col10", "date"),
+    }
+
+    data = {
+        "col1": "hello",
+        "col2": 1.0,
+        "col3": True,
+        "col4": datetime.datetime.now().replace(microsecond=0),
+        "col5": 10**10,
+        "col6": b"hello",
+        "col7": {"hello": "dave"},
+        "col8": Decimal(1),
+        "col9": 1,
+        "col10": datetime.date.today(),
+    }
+
+    with get_writer("parquet") as writer:
+        writer.write_data_item([data], columns)
+
+    with open(writer.closed_files[0], "rb") as f:
+        table = pq.read_table(f)
+        for key, value in data.items():
+            assert table.column(key).to_pylist() == [value]
+
+
+def test_parquet_writer_items_file_rotation() -> None:
+    columns = {
+        "col1": new_column("col1", "bigint"),
+    }
+
+    with get_writer("parquet", file_max_items=10) as writer:
+        for i in range(0, 100):
+            writer.write_data_item([{"col1": i}], columns)
+
+    assert len(writer.closed_files) == 10
+    with open(writer.closed_files[4], "rb") as f:
+        table = pq.read_table(f)
+        assert table.column("col1").to_pylist() == list(range(40, 50))
+
+
+def test_parquet_writer_size_file_rotation() -> None:
+    columns = {
+        "col1": new_column("col1", "bigint"),
+    }
+
+    with get_writer("parquet", file_max_bytes=2**8, buffer_max_items=2) as writer:
+        for i in range(0, 100):
+            writer.write_data_item([{"col1": i}], columns)
+
+    assert len(writer.closed_files) == 25
+    with open(writer.closed_files[4], "rb") as f:
+        table = pq.read_table(f)
+        assert table.column("col1").to_pylist() == list(range(16, 20))
+
+def test_parquet_writer_config() -> None:
+
+    os.environ["NORMALIZE__DATA_WRITER__VERSION"] = "2.0"
+    os.environ["NORMALIZE__DATA_WRITER__DATA_PAGE_SIZE"] =  str(1024 * 512)
+
+    with inject_section(ConfigSectionContext(pipeline_name=None, sections=("normalize", "data_writer",))):
```

Comment:
OK I get where the problem is coming from: look at this:
```
    class BufferedDataWriterConfiguration(BaseConfiguration):
        buffer_max_items: int = 5000
        file_max_items: Optional[int] = None
        file_max_bytes: Optional[int] = None
        disable_compression: bool = False
        _caps: Optional[DestinationCapabilitiesContext] = None

        __section__ = known_sections.DATA_WRITER
```
you need to add `__section__` to the config, not `section`. then you can drop "data_writer" from tuple of sections in the test. and it will work. good that we have this test